pyproject.toml

```diff
@@ -41,12 +41,12 @@ classifiers = [
 E8 = "flake8_eradicate:Checker"
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = "^3.7"
 
 flake8 = ">=3.5,<5"
 eradicate = "^2.0"
 attrs = "*"
-setuptools = "*"
+importlib-metadata = {version = "^4.11.3", python = "<3.8"}
```

Comment:
Let's use `*` for `importlib-metadata`. `"^4.11.3"` can cause problems for deps resolution.